crates/proof-of-sql/src/base/database/column_repetition_operation.rs

```diff
@@ -3,14 +3,18 @@ use super::{
     Column, ColumnType,
 };
 use crate::base::scalar::Scalar;
+use alloc::vec::Vec;
 use bumpalo::Bump;
-use core::iter::Iterator;
+use core::{iter, iter::Iterator};
 
-#[allow(dead_code)]
 pub trait RepetitionOp {
     fn op<T: Clone>(column: &[T], n: usize) -> impl Iterator<Item = T>;
 
+    // Special case for fixed-size binary columns.
+    fn op_fixed_size_binary(col_bytes: &[u8], width: usize, n: usize) -> Vec<u8>;
```

Comment:
Something _like_ this should work without needing to have a special case:
```suggestion
    fn op<T: Clone>(column: impl IntoIterator<T>, n: usize) -> impl Iterator<Item = T>;
```